src/main/java/org/pokemon/PokemonApp.java

```diff
@@ -18,14 +18,14 @@ public static void main(String[] args) {
         HighestAttackHandler highestAttackHandler = new HighestAttackHandler();
         PokemonStreamAcceptor pokemonStreamAcceptor = new PokemonStreamAcceptor();
         PokemonStreamHandler pokemonStreamHandler = new PokemonStreamHandler(
-                List.of(maxHpHandler,
+                Arrays.asList(maxHpHandler,
```

Comment:
Is there a specific reason you prefer Arrays.asList.   I just looked at the difference I prefer List.of - immutable, does not allow nulls – so appears more strict, I find all these nice qualities.  But List.of is only there from Java8, is that the reason?

Reply:
I now see your branch is named java 8 :) so this answers it.

Reply:
About the immutability, you are right. I think in Java 8, we can use guava for it. Or do this
List<String> stringList = Arrays.asList("a", "b", "c");
stringList = Collections.unmodifiableList(stringList);